run-cafmaker/run_cafmaker.sh

```diff
@@ -13,18 +13,28 @@ outFile=${tmpOutDir}/${outName}.CAF.root
 flatOutFile=${tmpOutDir}/${outName}.CAF.flat.root
 cfgFile=$(mktemp --suffix .cfg)
 
-./gen_cafmaker_cfg.py \
+# Compulsory arguments.
+args_gen_cafmaker_cfg=( \
     --base-dir "$ARCUBE_OUTDIR_BASE" \
-    --ghep-nu-name "$ARCUBE_GHEP_NU_NAME" \
-    --ghep-rock-name "$ARCUBE_GHEP_ROCK_NAME" \
-    --mlreco-name "$ARCUBE_MLRECO_NAME" \
-    --minerva-name "$ARCUBE_MINERVA_NAME" \
+    --spine-name "$ARCUBE_SPINE_NAME" \
+    --edepsim-name "$ARCUBE_SPILL_NAME" \
```

Comment:
In the idea of Edepsim being optional, I'd move that as an optional argument then